xml/depl_nodes.xml

```diff
@@ -5002,6 +5008,461 @@ getent group postgres | cut -d: -f3</screen>
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>num_of_logs</term>
+      <listitem>
+        <para>
+	  Maximum number of log entries the log agent sends to the &o_monitor;
+	  log API in a single request. Reducing the number increases performance.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>elapsed_time_sec</term>
+      <listitem>
+        <para>
+	  Time interval in seconds between sending logs to the &o_monitor; log API.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>delay</term>
+      <listitem>
+        <para>
+	  Interval in seconds for checking whether <literal>elapsed_time_sec</literal> has been reached. 
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>keystone</term>
+      <listitem>
+        <para>
+	  &o_ident; credentials the log agents use to send logs to the
+	  &o_monitor; log API. Do not change this option manually, as it is configured by &crow;.
+        </para>
+      </listitem>
+      </varlistentry>
+    </variablelist>
+
+    <bridgehead renderas="sect3"><guimenu>api: Settings for openstack-monasca-api</guimenu></bridgehead>
+
+    <variablelist>
+      <varlistentry>
+      <term>bind_host</term>
+      <listitem>
+        <para>
+	  Interfaces <literal>monasca-api</literal> listens on. Do not change
+	  this option, as it is configured by &crow;.
+       </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>processes</term>
+      <listitem>
+        <para>
+	  Number of processes to spawn.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>threads</term>
+      <listitem>
+        <para>
+	  Number of WSGI worker threads to spawn.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>log_level</term>
+      <listitem>
+        <para>
+	  Log level for <systemitem>openstack-monasca-api</systemitem>. Limits log messages to the specified level and
+	  above. The following levels are available: Critical, Error, Warning, Info
+	  (default), Debug, and Trace.
+        </para>
+      </listitem>
+      </varlistentry>
+    </variablelist>
+
+    <bridgehead renderas="sect3"><guimenu>elasticsearch: server-side settings for elasticsearch</guimenu></bridgehead>
+
+    <variablelist>
+      <varlistentry>
+      <term>repo_dir</term>
+      <listitem>
+        <para>
+	  List of directories for storing elasticsearch snapshots. Must be
+	  created manually and be writeable by the <systemitem
+	  class="username">elasticsearch</systemitem> user. 
+	  <!-- dpopov 2017-05-10:
+     FIXME: (Add reference the correct book/chapter) See the CMM operator's guide, section 3.4.1 for instructions on
+     creating an elasticsearch snapshot. --> Must contain at least one entry.
```

Comment:
No, it needn't contain any entries. By default it's empty, which means that there is no snapshot directory and snapshotting will not work.